zeebe/broker/src/main/java/io/camunda/zeebe/broker/system/configuration/backup/AzureBackupStoreConfig.java

```diff
@@ -124,6 +157,10 @@ public String toString() {
         + basePath
         + ", createContainer="
         + createContainer
+        + ", sasToken='"
+        + "<redacted>"
```

Comment:
:wrench: In many cases, while we don't want to show them, we would likely want to know if they're set. So I would propose simply not adding them to the `toString` when they are null. That will help when debugging. We can keep `<redacted>` when they are set ofc.